jmclient/jmclient/configure.py

```diff
@@ -185,10 +185,8 @@ def jm_single():
 # Only set to false for old wallets, Joinmarket is now segwit only.
 segwit = true
 
-# Use native segwit (bech32) wallet. This is NOT
-# currently supported in Joinmarket coinjoins. Only set to "true"
-# if specifically advised to do so.
-native = false
+# Use native segwit (bech32) wallet.
```

Comment:
Add a line to the comment to explain what happens if you set it to False (perhaps something like "if you set this to 'false', p2sh wrapped segwit (p2sh-p2wpkh) will be used when generating a new wallet; note that the default Joinmarket pit does not support this, and also note that pre-existing wallets do not change their type." ... well it's tricky to cover every detail, but these points seem like the most obvious ones to note.

Reply:
> note that the default Joinmarket pit does not support this

Making sure we are on the same page, do you mean that the default joinmarket pit does not support native segwit?

Reply:
@AdamISZ friendly ping!